pkg/sql/logictest/testdata/logic_test/zone_config

```diff
@@ -195,3 +195,22 @@ ORDER BY feature_name
 ----
 sql.schema.alter_range.configure_zone
 sql.schema.alter_table.configure_zone
+
+# Check entities for which we can set zone configs
+
+subtest test_entity_validity
+
+statement ok
+CREATE TABLE zc (
+  a INT PRIMARY KEY,
+  b INT
+)
+
+statement ok
+ALTER TABLE zc CONFIGURE ZONE USING gc.ttlseconds = 100000;
+
+statement ok
+CREATE MATERIALIZED VIEW vm(x,y) AS SELECT a,b FROM zc ; ALTER TABLE vm CONFIGURE ZONE USING gc.ttlseconds = 100000;
+
+statement error pgcode 42809 "v" is not a physical table
```

Comment:
could you also check for an error if trying to configure a virtual table, like `pg_catalog.pg_type` or `information_schema.columns`?

Reply:
Hey @rafiss I was trying to add test case for the virtual tables in pg_catalog, information_schema, and crdb_internal but the error I keep getting is 
"user root does not have ZONECONFIG or CREATE privilege on relation pg_type" .
So,  just wanted to ask is it allowed to configure these virtual  tables currently , or I am missing something

Reply:
Hey @rafiss thanks for your review , so I was trying to add test cases for virtual tables in virtual schemas like crdb_internal , pg_catalog and  information_schema but I keep getting privilege errors for the same .To be precise 
"user root does not have ZONECONFIG or CREATE privilege on relation session_variables" .

So just wanted to ask is there a way to allow configuring these virtual tables  even currently , or I am missing something

Reply:
oh nice find! then i think the test should just verify that the "does not have privilege" message is returned.